src/devices/SocketCan/CanRaw.cs

```diff
@@ -74,13 +74,16 @@ public bool TryReadFrame(Span<byte> data, out int frameLength, out CanId id)
             }
 
             CanFrame frame = new CanFrame();
-            
-            Span<CanFrame> frameSpan = MemoryMarshal.CreateSpan(ref frame, 1);
-            Span<byte> buff = MemoryMarshal.AsBytes(frameSpan);
-            while (buff.Length > 0)
+
+            unsafe
```

Comment:
Is MemoryMarshal not available in netstandard2.0?

Reply:
MemoryMarshal is, but not the CreateReadOnlySpan and CreateSpan